src/schoology-extractor/.vscode/settings.json

```diff
@@ -1,6 +1,5 @@
 {
     "python.analysis.extraPaths": ["./schoology_extractor", "./tests"],
-    "python.analysis.typeCheckingMode": "basic",
```

Comment:
Why remove that line?

Reply:
Pylance was showing a type error message for `parser.add()` method... I compared my settings.json file with the one from `lms-ds-loader` and I found that this line was not present in the loader. Removed it and it started working.

Reply:
Removing this actually disables the Pylance built-in type checking. I wonder if there is another way? Like in mypy.ini. I haven't found anything yet.

Can you see what happens when you restore this line, and then use `# type: ignore` on the import in that file that calls `parser.add()`

@bradbanister do you know of any way to get Pylance to ignore a 3rd party library without the inline ignore statement?

Reply:
@stephenfuqua I haven't found a good way yet, but haven't gotten around to looking yet.  I have a small number of similar issues but have been putting up with them.  Probably reaching critical mass soon though.

Reply:
`# type: ignore` works, but I need to add it each time that I call parser.add method, I am taking this approach for now.

Reply:
That's unfortunate.